datalad/support/annexrepo.py

```diff
@@ -2785,8 +2785,59 @@ def default_backends(self):
         else:
             return None
 
-    def fsck(self):
-        self._run_annex_command('fsck')
+    def fsck(self, paths=None, remote=None, fast=False, incremental=False,
+             limit=None, annex_options=None, git_options=None):
+        """Front-end for git-annex fsck
+
+        Parameters
+        ----------
+        paths : list
+          Limit operation to specific paths.
+        remote : str
+          If given, the identified remote will be fsck'ed instead of the
+          local repository.
+        fast : bool
+          If True, typically means that no actual content is being verified,
+          but tests are limited to the presence of files.
+        incremental : bool or {'continue'} or SCHEDULE
+          If given, `fsck` is called with `--incremental`. If 'continue',
+          `fsck` is additionally called with `--more`, and any other argument
+          is given to `--incremental-schedule`.
+        limit : str or all
+          If the functor `all` is given, `fsck` is called with `--all`. Any
```

Comment:
I'm guessing "functor" is a typo and you're not expecting readers to brush up on their category theory.

Reply:
Not a typo, this is how we talked about it in PyMVPA -- but I guess it could just be "function" ;-)

Reply:
Done